test/powershell/Modules/Microsoft.PowerShell.Utility/Select-Object.Tests.ps1

```diff
@@ -362,13 +340,48 @@ Describe "Select-Object with Property = '*'" -Tags "CI" {
         $obj.psobject.TypeNames.Count | Should -Be 3
         $obj.psobject.TypeNames[0] | Should -BeLike "Selected*"
         $obj.psobject.TypeNames[1] | Should -Not -BeLike "Selected*"
+        $p = Get-Process -Id $pid | Select-Object -Property Process* -ExcludeProperty ProcessorAffinity -ExpandProperty Modules
+        $p[0].psobject.Properties.Item("ProcessorAffinity") | Should -BeNullOrEmpty
```

Comment:
The new code should be moved after `$obj.psobject.TypeNames[2] | Should ..` line below.

Reply:
Looks like this got duplicated from above (L#328-329) somehow. Sorry about that, really not sure what on earth happened here. Will remove.